targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildAndroidOnLinux.sh.ejs

```diff
@@ -0,0 +1,40 @@
+!/bin/bash
+
+# echo ========== BUILDING ANDROID EXAMPLE PROJECT ==========
+usage="USAGE: BuildAndroidOnLinux.sh [archive directory]"
+
+if "%~1"=="" then USAGE()
+
+# Unreal Path, to change depending on your workspace setup.
+uePath=/Users/Shared/Epic Games
+
+# Unreal Version, to change on every unreal version
+ueVersion=<%- ueTargetVersion %>
+
+# Full Path. In theory it should not be edited (the engine path/version are the only ones that should be edited).
+uatPath=$uePath/UE_$ueVersion/Engine/Build/BatchFiles
+
+# Destination Path, where the build will end up (relative to active directory).
+archivePath=$1
```

Comment:
generally speaking, there are a lot of parameters to this script that are hard-coded.  any reason not to expose these as parameters to the script?